tests/integration_tests/nmap_integration_test.py

```diff
@@ -9,7 +9,7 @@
 @pytest.mark.integration
 @pytest.mark.asyncio
 async def test_integration():
-    timeout = 240
+    timeout = 900
```

Comment:
Comment saying yes an all port scan does take that long

Reply:
done this :)